docs/create_topology.md

```diff
@@ -234,7 +234,8 @@ node definitions interfaces, services, and initial configs can be specified.
 
 An example topology containing 3 Arista `cEOS` nodes and 2 Keysight `ixia-tg`
 ATEs can be found at
-[examples/3node-withtraffic.pb.txt](https://github.com/openconfig/kne/blob/df91c62eb7e2a1abbf0a803f5151dc365b6f61da/examples/3node-withtraffic.pb.txt).
+[examples/host/3node-host.pb.txt](https://github.com/openconfig/kne/blob/main/examples/host/3node-host.pb.txt).
```

Comment:
ceos-traffic is the topo that matches the description here: https://github.com/openconfig/kne/tree/main/examples/arista/ceos-traffic